app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -477,6 +484,30 @@ def parse_stack_resource(resource, group)
         return uid, new_result
       end
 
+      def parse_managed_image(image)
+        uid = image.id.downcase
+
+        new_result = {
+          :type               => ManageIQ::Providers::Azure::CloudManager::Template.name,
+          :uid_ems            => uid,
+          :ems_ref            => uid,
+          :name               => image.name,
+          :description        => "#{image.resource_group}\\#{image.name}",
+          :location           => @ems.provider_region,
+          :vendor             => 'azure',
+          :raw_power_state    => 'never',
+          :template           => true,
+          :publicly_available => false,
+          :operating_system   => process_os(image),
+          :hardware           => {
+            :bitness  => 64,
+            :guest_os => image.properties.storage_profile.try(:os_disk).try(:os_type) || 'unknown'
```

Comment:
right, didn't try the nil sent to normalize method, but the normalize method should be used, since we do infer the correct OS icons to show in the UI from this field.